src/components/StatusDot/StatusDot.enums.ts

```diff
@@ -0,0 +1,6 @@
+export const StatusDotColors = {
+  good: 'good',
+  bad: 'bad',
+  neutral: 'neutral',
+  inactive: 'inactive',
+} as const;
```

Comment:
Let's kill this file. I think the .enums files doesn't serve the purpose in our reality.